### ISSUE TITLE
bug: Javascript error occurred in the main process

### ISSUE THREAD

**Issue by madpink** [quanglam2807/singlebox#228]
# Bug Report
**Current Behavior:**

It seems somewhat random, but mostly occurs while reading GMail. Below is the actual text of the error popup.

```
A Javascript error occurred in the main process

Uncaught Exception:
TypeError: Object has been destroyed
    at WebContents.<anonymous> (/Applications/Singlebox.app/Contents/Resources/app.asar/build/libs/views.js:198:43)
    at WebContents.emit (events.js:210:5)
```

**Expected Behavior:**

No popup error.

**Steps to Reproduce:**

It seems random, it mostly occurs when interacting with GMail. The error may pop up several times in a five minute span, it may also be less than once an hour.

**Additional Context**

**Comment by vlad-cotfas** [comment]
Hmmmmm, I think I saw this error as a dialog when I did quit the SB app (CMD+Q (mac)).

**Comment by quanglam2807** [comment]
Thanks for reporting! The bug is fixed with #236. If it still occurs after the next update, please let me know
